src/command_modules/azure-cli-container/azure/cli/command_modules/container/custom.py

```diff
@@ -17,25 +17,28 @@
 from azure.cli.core._util import CLIError
 
 logger = _logging.get_az_logger(__name__)
-BASE_URL = "https://management.azure.com"
-RESOURCE_BASE_URL = "/subscriptions/{subscription_id}/resourceGroups/{resource_group_name}"
-CONTAINER_SERVICE_BASE_URL = RESOURCE_BASE_URL + "/providers/Microsoft.ContainerService"
-CONTAINER_SERVICE_RESOURCE_URL = (CONTAINER_SERVICE_BASE_URL +
-                                  "/containerServices/{container_service_name}")
-VISUAL_STUDIO_BASE_URL = RESOURCE_BASE_URL + "/providers/microsoft.visualstudio"
-VISUAL_STUDIO_ACCOUNT_URL = VISUAL_STUDIO_BASE_URL + "/account/{vsts_account_name}"
-VISUAL_STUDIO_PROJECT_URL = VISUAL_STUDIO_ACCOUNT_URL + "/project/{vsts_project_name}"
-RP_URL = BASE_URL + CONTAINER_SERVICE_RESOURCE_URL + "/providers/Microsoft.Mindaro"
+BASE_URL = "https://westus.mindaro.microsoft.io"
```

Comment:
Is this only available in westus? Does the base URL change for different cloud environments? As a preview this may not be relevant right now, but it will become relevant later.